### ISSUE TITLE
TextBlock hint issue on dark theme

### ISSUE THREAD

**Issue by kraxarn** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#1025]
When using the material design dark theme, hints on text blocks becomes difficult to read.
From the sample app:
![](https://vgy.me/MShCLT.png)
Is there any way to change the color it's using? I tried some different things, but couldn't figure it out (other than just changing the primary color, but that's not what I want).

**Comment by Keboo** [comment]
@kraxarn a simple option to address this is to simply override the theme brush that is being used in the TextBox resources. Something like this:
```XAML
<TextBox materialDesign:HintAssist.Hint="Floating Hint" 
         Style="{StaticResource MaterialDesignFloatingHintTextBox}">
    <TextBox.Resources>
        <SolidColorBrush x:Key="PrimaryHueMidBrush" Color="Fuchsia" />
    </TextBox.Resources>
</TextBox>
```

**Comment by Keboo** [comment]
Resolved by #1070. Use the new `HintAssist.Foreground` attached property to override the default theme colors for the hints.
